Proton/Sources/Base/AutogrowingTextView.swift

```diff
@@ -40,7 +40,7 @@ class AutogrowingTextView: UITextView {
     weak var boundsObserver: BoundsObserving?
     private var maxHeightConstraint: NSLayoutConstraint!
 
-    init(frame: CGRect, textContainer: NSTextContainer?, allowAutogrowing: Bool) {
+    init(frame: CGRect, textContainer: NSTextContainer?, allowAutogrowing: Bool = false) {
```

Comment:
I don't see why you introduced default value here and in RichTextView.

Reply:
To fix tests, otherwise I would have to change all places where it is being instantiated in tests.

Reply:
Because tests were broken before. Ok, clear 👍